### DISCUSSION TITLE
同一页第二张图片的坐标轴被加粗

### DISCUSSION THREAD

**Discussion by cymstrider** [sjtug/SJTUThesis#813] [提问] [answered]
本科论文模板里，添加的eps格式图片排版到一页之后，第二张图片的坐标轴也被加粗了，这个情况该怎么解决
![微信截图_20220523134633](https://user-images.githubusercontent.com/106053598/169751629-0083dc28-8ba0-4e63-af96-b3738fbb5c9e.jpg)

**Comment by skyzh** [comment]
这个是 xelatex 的伪粗体字体和 pdf 一起使用的 bug，不太好解决。可以安装 noto 系列字体并修改字体集 cjkfont=ubuntu；或者用 png 格式导入图片。

**Reply by cymstrider** [reply]
谢谢大佬，安装好的字体需要怎么导入项目吗？
“”修改字体集 cjkfont=ubuntu”应该在哪里完成？（抱歉是新手，不太会操作）

**Reply by skyzh** [reply]
https://github.com/sjtug/SJTUThesis/blob/b164d0f5b3087ed86c1038b5a85014b47a92c6f2/main.tex#L4-L10

`type=bachelor,cjk-font=ubuntu`

**Reply by skyzh** [reply]
可以参考 https://github.com/sjtug/SJTUThesis/wiki/在线使用说明

**Reply by cymstrider** [reply]
[https://github.com/sjtug/SJTUThesis/wiki/里需要的代码在/texmf/tex/latex/sjtuthesis/fd/sjtu-cjk-font-ubuntu.def文件里好像都给出了。怪不得直接用cjk-font=ubuntu，但是还是被报错了，是我字体安装的有问题吗：
![字体](https://user-images.githubusercontent.com/106053598/169777560-94d1456b-b40e-4957-af24-1a28a4b1518b.jpg)
![报错](https://user-images.githubusercontent.com/106053598/169777731-a6772ad7-22f8-4191-bcdf-d2196dbe5a0c.jpg)

**Reply by skyzh** [reply]
Noto 字体如果在本地安装，就只能在本地 TeXLive 使用。如果是 Overleaf，可以参照在线使用说明看看 Adobe 字体能不能解决问题？但总的来讲，还是建议嵌入一个 png。

**Reply by cymstrider** [reply]
太感谢了，用png解决了。
之前老师一直要求我插eps，刚刚用png发现图片质量也不差

**Comment by AlexaraWu** [comment, marked as answer]
插入一个白色字符就可以了，参见 [ustcthesis/issues/192](https://github.com/ustctug/ustcthesis/issues/192)。
